doc.cfm

```diff
@@ -3,6 +3,8 @@
 <cfif url.name IS "index">
 	<cfset data = {name="CFDocs", description="Ultra Fast CFML Documentation", type="index"}>
 <cfelseif FileExists(ExpandPath("./guides/en/#url.name#.md")) OR url.name is "how-to-contribute">
+
+	<cfset request.hasExamples = true>
```

Comment:
Is this line supposed to be here?

Reply:
yes you have an if statement added for the css and js files so only pages with examples load the prettify library

Reply:
since the verbiage made since "hasExamples" and since guides are in markdown so we can't guarantee (without some additional parsing/checking) there are in fact no examples on the page i felt this was the simplest approach

Reply:
ok makes sense thanks